rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -841,6 +854,31 @@ def test_exec(config, ssh_con):
                                     f"bucket check fix did not removed orphan objects on a bucket {bucket.name}"
                                 )
 
+                if config.test_ops.get("test_multipart_race_complete_abort"):
+                    utils.exec_shell_cmd(f"radosgw-admin gc process --include-all")
+                    log.info(
+                        "sleeping for 10 seconds before verifying objects download"
+                    )
+                    time.sleep(10)
+                    log.info("downloading all objects in bucket: %s" % bucket.name)
```

Comment:
@hmaheswa , can we use format in log messages where ever applicable

Reply:
there are many places in the code where this format exist @anrao19 , I just copied it from some other place, hence this problem..
PR review and merging is slowing down because of these to and fro for small things. my bad  :)
anyway updated where ever it exists in my new code @anrao19

Reply:
@hmaheswa , I had pointed this out in a previous comment also, but only one place was changed. So had to mention it again.
This is the usual check made in all the previous/future PRs. So request you to have this check in place for your future PRs